File: packages/ios/Sources/Components/WayfindingCard/WayfindingCard.figma.swift
import SwiftUI
import Figma

struct KozmosWayfindingCardConnect: FigmaConnect {
    let component = KozmosWayfindingCard<Text>.self
    let figmaNodeUrl = "https://figma.com/design/Yj4O8p6Y9h2Sa9zJVoAiVY?node-id=1340-6893"

    @FigmaString("Title Text")
    var title: String = "Navigation"

    @FigmaString("Body Text")
    var body_: String = "Route content"

    var body: some View {
        KozmosWayfindingCard(title: self.title, onClose: {}) {
            Text(self.body_)
        }
    }
}

File: packages/android/src/main/java/com/kozmos/components/WayfindingCard/WayfindingCard.figma.kt
package com.kozmos.components.wayfindingcard

import androidx.compose.runtime.Composable
import com.figma.code.connect.Figma
import com.figma.code.connect.FigmaConnect
import com.figma.code.connect.FigmaProperty
import com.figma.code.connect.FigmaType

@FigmaConnect("https://figma.com/design/Yj4O8p6Y9h2Sa9zJVoAiVY?node-id=1340-6893")
class KozmosWayfindingCardConnect {
    @FigmaProperty(FigmaType.Text, "Title Text")
    val title: String = "Navigation"

    @Composable
    fun ComponentExample() {
        KozmosWayfindingCard(title = title, onClose = {}) {}
    }
}
